src/core/Akka/Actor/ActorCell.DeathWatch.cs

```diff
@@ -4,13 +4,12 @@
 //     Copyright (C) 2013-2023 .NET Foundation <https://github.com/akkadotnet/akka.net>
 // </copyright>
 //-----------------------------------------------------------------------
-
+#nullable enable
```

Comment:
Been seeing some NRE's coming from the `Unwatch` handlers inside this class as part of the `ActorLeakSpec` - decided to enable `nullable` here to see what all is going on.